types/webidl-conversions/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"indent": [true, "tabs"]
```

Comment:
@sandersn has&nbsp;allowed me&nbsp;to&nbsp;set&nbsp;`indent` in&nbsp;`dtslint.json`:&nbsp;https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33478#issuecomment-470728638.

Reply:
Actually, now that we use prettier, I would prefer that all packages just use the default prettier settings.

Reply:
But I’ll still keep using tabs, because they are the superior choice and Prettier 2.0 might start using them by default.